src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/RepresentationUpdateHistory.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class RepresentationUpdateHistory {
```

Comment:
This class seems to be unnecessary, your choice, either refactor or create a techdebt and assign yourself to it due to time constraints